RTN-004.tex

```diff
@@ -53,7 +57,320 @@
+
+The application form will remain open, accumulating submissions, for the duration of DP0.
+Applications will be reviewed by the selection committee in two stages which coincide with the projected dates when the DP0.1 and DP0.2 science-ready data products will be made available to DP0 delegates in the RSP (Section \ref{ssec:intro_stages}).
+However, as described in Section \ref{ssec:del_resp}, delegates need not begin their DP0.1 participation at the time of account activation and may distribute their time over DP0 at their convenience.
+The DP0.1 data will be available to delegates who join with the second stage.
+
+To ensure that at least some accounts are available for new people to join for DP0.2, ~25\% of the 300 total RSP accounts will not be allocated until the second stage (DP0.2).
+The following paragraph provides additional motivation for this multi-stage approach to accounts allocation.
+
+The entire duration of DP0 is long, at least 1 year and potentially longer.
+It is anticipated that many delegates would not be able or willing to make time commitment for DP0.1 (minimal as it may be; Section \ref{ssec:del_resp}).
+Furthermore, some scientists might be more interested in the DP0.2 data set than DP0.1.
+Additionally, achieving diverse and equitable access to DP0 might also take some time: the advertisements must percolate, people must free up time in their schedules or obtain grants to fund their work, and the DP0-related resources and support network will become enriched over time (and thus more attractive to a wider variety of participants).
+To avoid the perception that access to any of DP0 is restricted to individuals who can commit to participating at the time of DP0.1, 25\% of the 300 RSP accounts will be reserved for DP0.2.
+Any accounts relinquished by DP0.1 participants will be added to this reservation and reallocated for DP0.2.
+
+Applications might also be reviewed on intermediate timescales in order to more fully include members of the groups identified in Table \ref{tab:delegate_groups} and achieve the diversity-related goals of DP0 participation.
+On such intermediate timescales the applications might be reviewed only by the Rubin Operations executive team and/or the Community Engagement Team instead of reconvening a full selection committee. 
+
+\begin{table}[!h]
+\centering
+\caption{Projected DP0 delegate application process timeline.}\label{tab:selection_timeline}
+\begin{tabular}{lllll}
+\hline
+Stage & \% (of 300) & Applications Due & Delegates Notified & Data Available \\
+\hline \hline
+DP0.1 & 75\%    &  Apr 30 2021   &  May 31 2021  &  Jun 30 2021 \\
+DP0.2 & 100\%  &  Jan 31 2022   &  Feb 28 2022  &  Mar 31 2022 \\
+\hline
+\end{tabular}
+\end{table}
+
+
+\section{Resources for DP0 Delegates}\label{sec:res}
+
+\subsection{Policies for RSP Accounts}\label{ssec:res_pol}
+
+For the duration of DP0, the RSP will be installed in the Interim Data Facility (IDF) deployed on the Google Cloud Platform.
```

Comment:
It should not matter that the IDF is deployed on Google Cloud Platform.

Reply:
Just giving the reader a bit of extra info, so I added a footnote link to the news item about the IDF.

For the duration of DP0, the RSP will be installed in the Interim Data Facility (IDF), which will be deployed on the Google Cloud Platform\footnote{\url{https://www.lsst.org/news/rubin-observatory-will-partner-google-cloud-host-interim-data-facility}}.